demos/browser/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "amazon-chime-sdk-js": "file:../..",
-    "aws-sdk": "^2.1105.0",
+    "aws-sdk": "^2.1152.0",
```

Comment:
Is this the actual change that enables it?

Reply:
yes, govcloud changes are already released by treb and tincan.